test-site/src/pages/PeoplePage.tsx

```diff
@@ -52,6 +52,18 @@ export function PeoplePage() {
             searchFields={filterSearchFields}
             searchOnSelect={true}
             label='Filters'
+            onDropdownInputChange={(value: string) => {
+              if (value !== "") return;
+              const fieldIds = filterSearchFields.map(field => field.fieldApiName);
+              const filtersToKeep: SelectableStaticFilter[] = [];
+              searchActions.state.filters.static?.forEach((staticFilter) => {
```

Comment:
so this use case is demonstrating removing static filters from the static filter state when they're used in filter search?

It might be helpful as documentation to explain a bit the intention behind this feature.